packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -235,8 +235,13 @@ const prompts = () => {
         },
         {
             name: 'siteId',
-            message: "What is your site's ID (examples: RefArch, SiteGenesis) in Business Manager?",
-            validate: validSiteId
+            message:
+                "What is your site's ID (examples: RefArchGlobal, SiteGenesis) in Business Manager?",
+            validate: validSiteId,
+            default: () => {
+                const defaults = testProjectAnswers()
```

Comment:
I get that we want to use the default values here, but using this function here is a bit off in this context to me. 🤔 . Also, the test project answer could be changed to be different in the future. I think it is best we have an object holding the default values instead of reusing them from this function.

Reply:
Thanks @alexvuong, this approach is what we're doing in the generator before removing that logic when we had the requirement to remove default answers.

You can see the same logic that we've been using for the longest time in the generator:
https://github.com/mobify/mobify-platform-sdks/commit/b43c8bdbb5d120952f1b211e4d5204adefebbc9e#diff-b40c3a39cfa0846c7c7d36515e883f7734adab8b527c9eb3aae0d14270a834a7L252-L256

PR removing default values: https://github.com/mobify/mobify-platform-sdks/pull/1548


The current approach has the advantage of having the value only in one place. If we ever need to change the default site Id we'll need to make changes only in one place, whereas if we create a new object with the default values apart from the `testProjectAnswers` values that contain the default answers, we'll need to change the value in two places. This is redundant to me. But I'm open to changing it if there's a good reason and the team prefers to have this value in a separate object.

Do you still think that it is worth it to create a separated object with this default value?

Reply:
Talking with Sejal, we decided to leave the changes on the generator for a future PR.